src/toil/test/src/jobTest.py

```diff
@@ -368,15 +368,15 @@ def runCheckpointVertexTest(self,
 
         # Run the workflow and check for the expected behavior
         options = Job.Runner.getDefaultOptions(self._getTestJobStorePath())
-        options.logLevel = "INFO"
+        options.logLevel = "DEBUG"
```

Comment:
Same as last comment

Reply:
I like leaving it in, but `info` or `critical` will reduce the logs it produces.  Don't set this to `debug`.